.github/workflows/copy-workflow.yml

```diff
@@ -62,109 +58,54 @@ jobs:
       run: |
         git config user.name ${{ env.GITHUB_USER }}
         git config user.email ${{ env.GITHUB_EMAIL }}
-    - name: is initial test workflow deployment
+    - name: determine files to add
       run: |
-        if [[ ! -f $TARGET_REPO_DIR/.github/workflows/go-test.yml ]]; then
-          echo "INITIAL_TEST_DEPLOYMENT=1" >> $GITHUB_ENV
-        fi
-    - name: remove Travis (on initial deployment)
-      if: ${{ env.INITIAL_TEST_DEPLOYMENT == 1 }}
+        files=${{ toJson(toJson(matrix.cfg.files)) }}
+        files=$(echo -e "$files" | jq -c '.')
+        echo "FILES=$files" >> $GITHUB_ENV
+    - name: is initial workflow deployment
+      run: |
+        INITIAL_WORKFLOW_DEPLOYMENT=1
+        for f in $(jq -r '.[]' <<< $FILES); do
+          if [[ -f $TARGET_REPO_DIR/$f ]]; then
+            INITIAL_WORKFLOW_DEPLOYMENT=0
+            break
+          fi
+        done
+        echo "INITIAL_WORKFLOW_DEPLOYMENT=$INITIAL_WORKFLOW_DEPLOYMENT" >> $GITHUB_ENV
```

Comment:
`INITIAL_WORKFLOW_DEPLOYMENT` is going to be `1` iff none of the files in the target repository exist yet.

Reply:
Can you add this as a comment?